### ISSUE TITLE
Jest 28 error 'invalid return value' with process() or/and processAsync()

### ISSUE THREAD

**Issue by jafin** [routablehq/jest-scss-transform#5]
**Describe the bug**

Using jest-scss-transform with Jest 28 produces an error:  (ok in jest 27)

```shell
 Invalid return value:
      `process()` or/and `processAsync()` method of code transformer found at
      "node_modules/jest-scss-transform/lib/index.js"
      should return an object or a Promise resolving to an object. The object
      must have `code` property with a string of processed code.
      This error may be caused by a breaking change in Jest 28:
      https://jestjs.io/docs/upgrading-to-jest28#transformer
      Code Transformation Documentation:
      https://jestjs.io/docs/code-transformation
```

Possibly relates to a change in jest v28

>`process()` and `processAsync()` methods of a custom [transformer module](https://jestjs.io/docs/code-transformation) cannot return a string anymore. They must always return an object:

```js
 process(sourceText, sourcePath, options) {
-   return `module.exports = ${JSON.stringify(path.basename(sourcePath))};`;
+   return {
+     code: `module.exports = ${JSON.stringify(path.basename(sourcePath))};`,
+   };
  }
```

**To Reproduce**

1. Configure jest with jest-scss-transform.
2. Use jest 28

**Expected behavior**

Jest runs as expected

**Environment (please complete the following information):**

- Windows 11 21H2 22000.613
- jest-scc-transform v1.0.2
- Jest v28.0.3
- Node v16.14.2

**Comment by ghasemikasra39** [comment]
I have the same issue in `"react-native": "0.68.2"` with `"jest": "^28.1.0"`. Any solution?

**Comment by zigcccc** [comment]
Hey @jafin thanks for reporting this! We are investing the described issue & are hoping to resolve it ASAP.

**Comment by zigcccc** [comment]
Hey @jafin, we have just released [a new version 1.0.3](https://www.npmjs.com/package/jest-scss-transform/v/1.0.3) which should address the described issue for jest v28 and above.

That should get you going now. Again, appreciate you reporting this & don't hesitate to circle back if you're still seeing any issues!
